app/controllers/application_controller.rb

```diff
@@ -23,6 +20,11 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def multisite_install
```

Comment:
this method might look better as `multisite_install?` - at first i thought it was going perform some sort of install